src/sentry/snuba/metrics/extraction.py

```diff
@@ -224,11 +228,12 @@ def get_default_spec_version(cls: Any) -> SpecVersion:
     "p95": "p95",
     "p99": "p99",
     # p100 is not supported in the metrics layer, so we convert to max which is equivalent.
-    "p100": "max"
+    "p100": "max",
     # generic percentile is not supported by metrics layer.
 }
 
 # Maps plain Discover functions to derived metric functions which are understood by the metrics layer.
+# XXX: We need to support count_miserable
```

Comment:
For expediency's sake, I decided not to support it at https://github.com/getsentry/sentry/issues/58233